### PR TITLE
feat: added cli functionality to dataproc quickstart sample

### PR DESCRIPTION
**Pull request by bradmiro**
I changed the formatting of the dataproc quickstart sample to make it an end-to-end tool for devs to use with CLI functionality via flags. In discussing with the TW we felt the entire script should be exposed given that this example is end-to-end.

### CODE REVIEW

File: samples/quickstart.js [hunk 1]
```diff
@@ -142,7 +145,30 @@ function main(projectId, region, clusterName, jobFilePath) {
...
-main(...process.argv.slice(2));
+const args = require('yargs').options({
+  project_id: {
+    demandOption: true,
+    describe: 'Project to use for creating resources.',
+    type: 'string',
+  },
+  region: {
+    demandOption: true,
+    describe: 'Region where the resources should live.',
```
Comment (bcoe) [inline samples/quickstart.js]: I agree that in a standalone application this is an improvement (I actually am one of the main folks who's worked on yargs throughout the years, and love it), but @fhinkel's sample guidelines call for samples to be imbedable without any dependencies; which I think is a reasonable tradeoff.

tldr; we probably shouldn't introduce yargs.

Comment (bradmiro) [reply samples/quickstart.js]: Removed yargs dependency
